next.config.js

```diff
@@ -1,5 +1,7 @@
 /** @type {import('next').NextConfig} */
 
+const { env } = require("process");
```

Comment:
Is this needed?

Reply:
No, I removed it. There is still a bug in the eslint and prettier config. It does not yet properly remove not needed imports. Or only in some cases. It's something I still have to figure out how to solve this.